src/appengine/private/components/upload-testcase/upload-form-simplified.html

```diff
@@ -146,11 +146,11 @@
         </template>
       </div>
       <div class="warning">
-        <b>WARNING</b>: ClusterFuzz doesn't support untrusted workloads.<br>
-        Only upload testcases that you trust enough to run on your own
-        machine. Note that the following jobs do use the Chrome sandbox
-        but this is not a guarantee of protection:<br>
-        [[formatSandboxedJobs(fieldValues.sandboxedJobs)]]
+	      <span><strong>WARNING</strong>: ClusterFuzz only partially supports untrusted workloads.<br></span>
+        ClusterFuzz supports untrusted workloads when the platform is Linux <br>
```

Comment:
nit: can we get rid of the extra space before <br>

Reply:
yeah idk why that happens bc now it looks weird on my editor - but fixed!!